### DISCUSSION TITLE
Certificate Not Trusted in Chrome, but Works in Safari

### DISCUSSION THREAD

**Discussion by michaelgriffindrake** [cockpit-project/cockpit#19532] [Q&A]
### Explain what happens

Hey there!

Thanks for developing cockpit, I'm liking it so far. I'm having an issue with the SSL certificate. I exported it from chrome (when it complained it wasn't safe) and added it to my keychain in Mac. The site now loads properly in Safari, but still says not trusted in Chrome. What gives? Any ideas on a fix?

<img width="502" alt="Screenshot 2023-10-25 at 7 23 12 PM" src="https://github.com/cockpit-project/cockpit/assets/1414868/0c9c5d58-4671-422e-b9ed-144e1838406f">
<img width="170" alt="Screenshot 2023-10-25 at 7 25 21 PM" src="https://github.com/cockpit-project/cockpit/assets/1414868/8c318463-eb44-4ba4-84b0-4a4282b5c661">

### Version of Cockpit

 300.1-1~bpo22.04.1

### Where is the problem in Cockpit?

None

### Server operating system

other

### Server operating system version

Linux Mint 21.2

### What browsers are you using?

Chrome, Safari macOS

**Comment by martinpitt** [comment]
If you *explicitly* import the certificate as trusted CA in Safari, then yes, it will work. You can do the same in Firefox, I'm sure you can also do this in Chrome. However, it is correct -- if you talk to a remote machine, the auto-generated self-signed certificate really is not trustworthy inherently. If you have a real certificate, please [install it for cockpit](https://cockpit-project.org/guide/latest/https.html#https-certificates), set up [LetsEncrypt](https://github.com/cockpit-project/cockpit/wiki/Proxying-Cockpit-over-Apache-with-LetsEncrypt), or if you only talk to `localhost`, use the unencrypted URL: http://localhost:9090.
